lib/project_types/script/commands/deploy.rb

```diff
@@ -9,6 +9,9 @@ class Deploy < ShopifyCli::Command
       end
 
       def call(args, _name)
+        opt = CLI::UI::Prompt.confirm(@ctx.message('script.deploy.replace_warning'))
+        return unless opt
```

Comment:
I am also NOT sure if we should warn about replacing, if this is the very first app script that is being deployed?  Would this be more confusing?  Maybe another approach is also to display a Tip after the command finishes successfully?

Reply:
@ErinRenNumber1 this is where my limitations of what I know about Scripts is bleeding through. My assumption was that we would only show the tip if a script was already deployed once before. The first time a script is deployed, we shouldn't show it.